docs/NEW_NEAR_CLI_INTERFACE.md

```diff
@@ -42,15 +42,18 @@ account
   - list-keys <account-id> network <"mainnet"|"testnet"|...> <now|at-timestamp|at-block-height|at-block-hash>
 
   - add-key <account-id>
-    - autogenerate-new-keypair network <"mainnet"|"testnet"|...>
-      - save-to-keychain network <"mainnet"|"testnet"|...>
+    - grant-full-access
+      - autogenerate-new-keypair
+        - save-to-keychain network <"mainnet"|"testnet"|...>
+          - transaction signature options here (see below)
+        - print-to-terminal network <"mainnet"|"testnet"|...>
+          - transaction signature options here (see below)
+      - use-manually-provided-seed-phrase "twelve words goes here" network <"mainnet"|"testnet"|...>
```

Comment:
This is an issue I filled with near-cli.  It's bad security practice to use a seed phrase as a CLI arg. So perhaps this should be interactive like a password.

Reply:
It is both interactive and non-interactive, and again, I don’t expect users to type the command out manually most of the time, so the parameters are there only for reproducibility and integration within scripts, which I don’t see a reason to limit to use params